Cargo.toml

```diff
@@ -9,3 +9,4 @@ authors = ["Mark Goodwin <mgoodwin@mozilla.com>", "Dana Keeler <dkeeler@mozilla.
 byteorder="1.3.1"
```

Comment:
Do you want to add your name to the authors? (I can't seem to comment on that line...)
Also, I imagine we should bump the version.

Reply:
Sure, I'll do that in the next commit. Thanks